core/src/main/java/fr/sncf/osrd/infra/implementation/reservation/ReservationInfraBuilder.java

```diff
@@ -148,11 +149,11 @@ private ImmutableNetwork<DiDetector, ReservationRoute> makeRouteGraph(
 
     /** Checks that the route makes sense, reports any relevant warning or error otherwise */
     private void validateRoute(ReservationRouteImpl route, RJSRoute rjsRoute) {
-        if (route.getDetectorPath().isEmpty()) {
+        if (route.getDetectorPath().size() < 2) {
```

Comment:
what about using constant for that property that needs to be reused in unit tests ?

Reply:
it's not a magic number, it's a basic property of routes: they have an entry and exit detector